### ISSUE TITLE
NVDA doesn't recognize: ttype="text" with placeholder

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1638]
**Reported by kevinchao89 on 2011-07-02 15:24**
Firefox 7.0a1 (2011-07-02), NVDA Snapshot 4523

http://www.html5accessibility.com/tests/form-test.html
Navigate page by form fields until you reach: 
type="text" with placeholder 
Expected: For NVDA to indicate placeholder, which is "some text". Placeholder should be cleared when activating focus mode or typing text into field.
Actual: NVDA treats edit box as a regular edit field, which has an autocomplete.

**Comment by dkager** [comment]
How does this related to #7004?

**Comment by jcsteh** [comment]
After #7004, NVDA now supports HTML placeholder and aria-placeholder in Chrome. In Firefox, aria-placeholder is supported, but not HTML placeholder. The latter requires https://bugzilla.mozilla.org/show_bug.cgi?id=1303429.

Closing as duplicate.
